sale_project_milestone/models/project_milestone.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ProjectMilestone(models.Model):
+
+    _inherit = "project.milestone"
+
+    sale_line_id = fields.Many2one(copy=False, index=True)
```

Comment:
Hello @lanto-razafindrabe 

Field already existe in this object see [here](https://github.com/odoo/odoo/blob/16.0/addons/sale_project/models/project_milestone.py#L13)